odoo/tests/common.py

```diff
@@ -348,6 +348,22 @@ def __init__(self, methodName='runTest'):
         self.addTypeEqualityFunc(etree._Element, self.assertTreesEqual)
         self.addTypeEqualityFunc(html.HtmlElement, self.assertTreesEqual)
 
+    def run(self, result):
+        ODOO_TEST_FAILURE_RETRIES = int(os.environ.get('ODOO_TEST_FAILURE_RETRIES', 1))
+        failure = False
+        for retry in range(ODOO_TEST_FAILURE_RETRIES):
+            if retry == 0 or failure:
```

Comment:
This seems not useful: if `not failure` then the loop is broken out of, so we should only be able to reach `retry != 0` if `failure`.